macros/InterfaceData.json

```diff
@@ -1275,6 +1287,14 @@
     "inh"  : "Event",
     "impl" : []
   },
+  "RTCIdentityProviderGlobalScope": {
```

Comment:
RTCIdentityGlobalScope isn't implemented at this time, true. I'll remove it for now.